### ISSUE TITLE
Question: How to change the design of a form

### ISSUE THREAD

**Issue by rabol** [kdion4891/laravel-livewire-forms#6]
Let's say that I want a row with first name, middle and last name or input groups to have icons or text in front or after the input (https://www.w3schools.com/bootstrap4/bootstrap_forms_input_group.asp)

Thanks

**Comment by rabol** [comment]
I have figured out the input groups - it can be handled with adding ->view()

**Comment by kejojedi** [comment]
correct. you can also publish the package views with `php artisan vendor:publish --tag=form-views`
